app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\CategoryRequest;
+use App\Models\Category;
+
+class CategoryController extends Controller
+{
+    public function index()
+    {
+        $categories = Category::with('children')->whereNull('parent_id')->get();
+        return view('admin.pages.category.index', compact('categories'));
+    }
+
+    public function add()
+    {
+        $categories = Category::select('id', 'name', 'description', 'parent_id')->get();
```

Comment:
why this required?

Reply:
parent category list show karva mate

Reply:
jarur nathi. tamare dropdown ma select nathi karvani